rholang/src/main/scala/coop/rchain/rholang/interpreter/dispatch.scala

```diff
@@ -51,16 +51,15 @@ class RholangAndScalaDispatcher[M[_]] private (
 
 object RholangAndScalaDispatcher {
 
-  def create[M[_], F[_]](
+  def create[M[_]: _error, F[_]](
```

Comment:
I'm aware that the code base is riddled with bad examples like this - but I have to say it - this is not an appropriate use of type-classes, `Ref` is not a capability or constraint - it's data.

Reply:
I agree. I'm merely replacing a bad example `ErrorLog` with another bad example `_error`.